authors/apps/articles/views.py

```diff
@@ -500,3 +504,42 @@ def send_mail(sender_email, receiver_mail, mail_subject, content):
     mail = Mail(from_email, subject, to_email, content)
     response = sg.client.mail.send.post(request_body=mail.get())
     return response
+class RateView(APIView):
+    permission_classes = (IsAuthenticated,)
+    serializer_class = RateSerializer
+
+    def average_rating(self, article):
+
```

Comment:
Please remove this redundant line.

Reply:
Thanks for pointing that out, I have worked on it!